modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItCommonApiTest.java

```diff
@@ -123,37 +115,25 @@ public void testSessionExpiration() throws Exception {
     @Test
     public void checkTimestampOperations() {
         String kvTblName = "tbl_all_columns_sql";
-        String schemaName = "PUBLIC";
-        String keyCol = "key";
-        int maxTimePrecision = TemporalColumnType.MAX_TIME_PRECISION;
+        String keyCol = "KEY";
 
         Ignite node = CLUSTER_NODES.get(0);
 
         // TODO: https://issues.apache.org/jira/browse/IGNITE-19162 Trim all less than millisecond information from timestamp
         //String tsStr = "2023-03-29T08:22:33.005007Z";
-        String tsStr = "2023-03-29T08:22:33.005Z";
+        String tsStr = "2023-03-29T08:22:33Z";
 
         Instant ins = Instant.parse(tsStr);
 
         sql("CREATE TABLE timestamps(id INTEGER PRIMARY KEY, i TIMESTAMP(9))");
 
-        TableDefinition schTblAllSql = SchemaBuilders.tableBuilder(schemaName, kvTblName).columns(
-                SchemaBuilders.column(keyCol, ColumnType.INT64).build(),
-                SchemaBuilders.column("time", ColumnType.time(maxTimePrecision)).asNullable(true).build(),
-                SchemaBuilders.column("timestamp", ColumnType.timestamp(maxTimePrecision)).asNullable(true).build(),
-                SchemaBuilders.column("datetime", ColumnType.datetime(maxTimePrecision)).asNullable(true).build()
-        ).withPrimaryKey(keyCol).build();
-
-        await(((TableManager) node.tables()).createTableAsync(schTblAllSql.name(), DEFAULT_ZONE_NAME, tblCh ->
-                SchemaConfigurationConverter.convert(schTblAllSql, tblCh)
-        ));
+        sql(String.format("CREATE TABLE %s(\"%s\" INTEGER PRIMARY KEY, \"TIMESTAMP\" TIMESTAMP(9))", kvTblName, keyCol));
```

Comment:
Why do you drop DATETIME column?

Reply:
There is a nuance here, here I left `org.apache.ignite.sql.ColumnType#DATETIME` which in SQL has the `org.apache.calcite.sql.type.SqlTypeName#TIMESTAMP`, but I can’t use `org.apache.ignite.sql.ColumnType#TIMESTAMP` because in SQL it is type `org.apache.calcite.sql.type.SqlTypeName#TIMESTAMP_WITH_LOCAL_TIME_ZONE` which does not currently work here is the IGNITE-17376 and IGNITE-19274.

I looked at the mapping in `org.apache.ignite.internal.sql.engine.util.TypeUtils#columnType`.

Reply:
Ok. Let's add these tickets to comment as well, or disable the test with these tickets numbers.